src/components/EntityInfo/EntityMenu.vue

```diff
@@ -70,7 +70,7 @@
     </span>
 
     <!-- More Actions -->
-    <span v-if="isBusiness && !isHistorical">
+    <span v-if="isBusiness && (!isHistorical || isAllowed(AllowableActions.DIGITAL_CREDENTIALS))">
```

Comment:
It looks like you're trying to prevent an empty sub-menu. When can this happen?

I recommend that you extract these conditions into a local getter (computed) below, where you can add comments explaining the logic.

Reply:
That's basically it. It will display if a company was historical but didn't have the DC feature available. Have added a getter to explain that

Reply:
LGTM